scripts/eslint.py

```diff
@@ -0,0 +1,75 @@
+"""  # pylint: disable=django-not-configured
+Check code quality using pycodestyle, pylint, and diff_quality.
+"""
+
+import re
+import subprocess
+import shlex
+import sys
+
+
+class BuildFailure(Exception):
+    pass
+
+
+def fail_quality(name, message):
```

Comment:
The value of `name` in this script is always `"eslint"`, so there is no reason to take it as an argument. Just hard-code `"xsslint`" as the name. Be simple.